### ISSUE TITLE
tileClassName working on Chrome not Safari

### ISSUE THREAD

**Issue by ncevallos** [wojtekmaj/react-calendar#551]
I spent an hour looking into this locally thinking that it was just a different Safari CSS interpretation that was creating the issue, but once I got into the console on Safari, I've learn that the react-calendar is not appending the custom classes in Safari as it is in Chrome. Has anyone encountered this issue? And if so what is the fix/work-around?
The first screenshot shows the Chrome console with custom classes appended.
![Screen Shot 2021-10-03 at 3 59 49 PM](https://user-images.githubusercontent.com/26979403/135769695-d9ac9b61-172d-48e8-95e1-dcf5f6fcf03f.png)
The second screenshot shows the Safari console with no custom classes.
![Screen Shot 2021-10-03 at 4 02 27 PM](https://user-images.githubusercontent.com/26979403/135769696-422cb85e-8cbb-4ab6-bc02-b2c0b592d98d.png)
Any advice on what I might be doing wrong would be greatly appreciate. :)

**Comment by wojtekmaj** [comment]
That seems super weird! Can you share your `tileClassName` function? Did you make sure that it returns the same value on both browsers?

**Comment by ncevallos** [comment]
<img width="665" alt="Screen Shot 2021-10-03 at 8 00 11 PM" src="https://user-images.githubusercontent.com/26979403/135777877-05dcca51-4c8e-4ba5-af0f-f44450e5152d.png">
This is the tileClassName function. The return value shouldn't matter which browser right? I just noticed like I had shared in the previous images, that the desired classNames only get appended in Chrome.

**Comment by DamienLucchese** [comment]
Howdy! Just checking if there's been any movement / additional insight. I'm also seeing this issue in Safari & Firefox. While debugging, I realized the buttons were not receiving the classes being sent into the component...but only in Safari & Firefox. How odd. Thank you. It's such a great package, I would love to be able to continue using it in my project.

**Comment by RWHaines** [comment]
I had this exact same issue.  Chrome and Edge worked perfectly fine but Firefox and Safari did not.  What I changed was how the date object was formatted.  I switched my `-` characters to `/` in the date format and the tiles were highlighted properly after that.  Your function may be evaluated differently across browsers so it would be helpful for you to double check that the function is evaluating how you expect it to.
